src/lib.rs

```diff
@@ -37,7 +37,10 @@ mod throughput;
 #[cfg(feature = "tester")]
 pub use self::throughput::ThroughputMonitoring;
 
+#[cfg(test)]
+pub mod test_utils;
+
 pub use self::config::Config;
 pub use self::error::{ErrorKind, Result};
-pub use self::net::{LinkConditioner, Socket, SocketEvent};
```

Comment:
When users want to simulate packet drop, they should be able to do so. Maybe not within the public API but behind a feature. Where is this `LinkConditioner` moved to?

Reply:
`LinkConditioner` was moved to `test_utils/link_conditioner.rs`.
I like the idea to hide it behind the feature flag, it makes more sense.